msal/src/main/java/com/microsoft/identity/client/SingleAccountPublicClientApplication.java

```diff
@@ -92,63 +88,43 @@ private void initializeSharedPreferenceFileManager(@NonNull final Context contex
-                            mPublicClientConfiguration.getAppContext(),
-                            params.getAuthority(),
-                            mPublicClientConfiguration
-                    ),
-                    new CommandCallback<List<ICacheRecord>, BaseException>() {
-                        @Override
-                        public void onTaskCompleted(final List<ICacheRecord> result) {
-                            // To simplify the logic, if more than one account is returned, the first account will be picked.
-                            // We do not support switching from MULTIPLE to SINGLE.
-                            // See getAccountFromICacheRecordList() for more details.
-                            checkCurrentAccountNotifyCallback(callback, result);
-                        }
-
-                        @Override
-                        public void onError(final BaseException exception) {
-                            com.microsoft.identity.common.internal.logging.Logger.error(
-                                    TAG + methodName,
-                                    exception.getMessage(),
-                                    exception
-                            );
-
-                            callback.onError(MsalExceptionAdapter.msalExceptionFromBaseException(exception));
-                        }
-
-                        @Override
-                        public void onCancel() {
-                            //Do nothing
-                        }
+        final GetCurrentAccountCommand command = new GetCurrentAccountCommand(
+                params,
+                controller,
+                new CommandCallback<List<ICacheRecord>, BaseException>() {
+                    @Override
+                    public void onTaskCompleted(final List<ICacheRecord> result) {
+                        // To simplify the logic, if more than one account is returned, the first account will be picked.
+                        // We do not support switching from MULTIPLE to SINGLE.
+                        // See getAccountFromICacheRecordList() for more details.
+                        checkCurrentAccountNotifyCallback(callback, result);
                     }
 
+                    @Override
+                    public void onError(final BaseException exception) {
+                        callback.onError(MsalExceptionAdapter.msalExceptionFromBaseException(exception));
+                    }
 
-            );
-
-            loadAccountCommand.setPublicApiId(PublicApiId.GET_CURRENT_ACCOUNT_ASYNC);
-            CommandDispatcher.submitSilent(loadAccountCommand);
+                    @Override
+                    public void onCancel() {
+                        //Do nothing
+                    }
+                });
 
-        } catch (MsalClientException clientException) {
-            callback.onError(clientException);
-        }
+        command.setPublicApiId(PublicApiId.GET_CURRENT_ACCOUNT_ASYNC);
```

Comment:
Update your branch with `dev` and then you'll need to use the `SINGLE_ACCOUNT_PCA_GET_CURRENT_ACCOUNT_ASYNC` api id here